packages/gatsby-theme-docs/src/layouts/release-notes-list.js

```diff
@@ -13,7 +13,7 @@ import LayoutFooter from './internals/layout-footer';
 import LayoutPageWrapper from './internals/layout-page-wrapper';
 import LayoutPage from './internals/layout-page';
 import LayoutPageHeader from './internals/layout-page-header';
-import LayoutPageHeaderSide from '../overrides/layout-page-header-side';
+import LayoutReleaseNotePageHeaderSide from './layout-release-note-page-header-side';
```

Comment:
Yes it should. But for now we don't want to override the release note layout. If we decide in the future that we want to override that as well, we can easily change it.
I think we should make overrides only available if we really need to use it somewhere. Otherwise it gets more and more confusing...also with the documentation.

Reply:
ok, thank you 👍